.travis.yml

```diff
@@ -0,0 +1,21 @@
+script:
+  - pushd source
+  - make -j2
```

Comment:
@tkoeppe I'll give `latexmk` a try. I think I'll have to duplicate some of the effort from the `Makefile` to the `.travis.xml` file (e.g., building the diagram PDFs with `dot`, generating the grammar and cross-references appendices) since `latexmk` won't do that for us.

Is it worth the effort or is it better to rely on the `Makefile`?